agent/agent-tooling/src/main/java/com/microsoft/applicationinsights/agent/internal/exporter/AzureMonitorMetricsData.java

```diff
@@ -0,0 +1,92 @@
+/*
+ * ApplicationInsights-Java
+ * Copyright (c) Microsoft Corporation
+ * All rights reserved.
+ *
+ * MIT License
+ * Permission is hereby granted, free of charge, to any person obtaining a copy of this
+ * software and associated documentation files (the ""Software""), to deal in the Software
+ * without restriction, including without limitation the rights to use, copy, modify, merge,
+ * publish, distribute, sublicense, and/or sell copies of the Software, and to permit
+ * persons to whom the Software is furnished to do so, subject to the following conditions:
+ * The above copyright notice and this permission notice shall be included in all copies or
+ * substantial portions of the Software.
+ * THE SOFTWARE IS PROVIDED *AS IS*, WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED,
+ * INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR
+ * PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE
+ * FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR
+ * OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER
+ * DEALINGS IN THE SOFTWARE.
+ */
+
+package com.microsoft.applicationinsights.agent.internal.exporter;
+
+import static io.opentelemetry.api.internal.Utils.checkArgument;
+
+import com.azure.monitor.opentelemetry.exporter.implementation.builders.MetricPointBuilder;
+import com.azure.monitor.opentelemetry.exporter.implementation.builders.MetricTelemetryBuilder;
+import com.azure.monitor.opentelemetry.exporter.implementation.models.DataPointType;
+import io.opentelemetry.sdk.metrics.data.DoublePointData;
+import io.opentelemetry.sdk.metrics.data.HistogramPointData;
+import io.opentelemetry.sdk.metrics.data.LongPointData;
+import io.opentelemetry.sdk.metrics.data.MetricData;
+import io.opentelemetry.sdk.metrics.data.MetricDataType;
+import io.opentelemetry.sdk.metrics.data.PointData;
+
+final class AzureMonitorMetricsData {
+
+  private static final String DEFAULT_METRIC_AGGREGATION_INTERVAL = "60000";
```

Comment:
can we get the interval from the metric data somehow? in the agent we could use the configured metric interval since we configure that, but for using the metric exporter standalone, we don't have control and users could set the reader interval to anything

Reply:
interval is set at the reader level.. our exporter doesn't have that info.. metricData doesn't have that either.

Reply:
can you propose a feature request to upstream to make that property available in MetricData?

Reply:
I'm not sure that it's not already available, what about MetricData `epochNanos - startEpochNanos`, I'm thinking that may be the interval

Reply:
can you test with epochNanos - startEpochNanos to see if it works?

Reply:
getting the interval right is important, so I'd like to at least try to get it right in this PR if possible

Reply:
i was able to confirm epochNanos - startEpochNanos is the interval. i tested it with different intervals.. result is persisting.